scripts/DeployCore.s.sol

```diff
@@ -3,31 +3,35 @@ pragma solidity ^0.8.17;
 
 import "../lib/forge-std/src/Script.sol";
 
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+
 import {ConfigLib} from "../lib/ConfigLib.sol";
 import {CheckLib} from "../lib/CheckLib.sol";
 import {DeployLib} from "../lib/DeployLib.sol";
 
 contract DeployCore is Script {
-    using ConfigLib for ConfigLib.HyperlaneDomainConfig;
-    using CheckLib for ConfigLib.HyperlaneDomainConfig;
-    using DeployLib for ConfigLib.HyperlaneDomainConfig;
+    using Strings for uint256;
 
-    function run() public {
-        string memory local = vm.envString("LOCAL");
-        string[] memory remotes = vm.envString("REMOTES", ",");
-        ConfigLib.HyperlaneDomainConfig memory config = ConfigLib
-            .readHyperlaneDomainConfig(vm, local);
-        ConfigLib.MultisigIsmConfig memory ismConfig = ConfigLib
-            .readMultisigIsmConfig(vm, remotes);
+    using DeployLib for ConfigLib.Core;
+    using ConfigLib for ConfigLib.Core;
+    using CheckLib for DeployLib.Core;
 
-        vm.startBroadcast();
-        uint256 startBlock = block.number;
+    function run() public returns (DeployLib.Core memory core) {
```

Comment:
Note you'll need to modify the docs instructions now that multisig deployment does not happen as part of this (among other changes)